src/main/drivers/timer_stm32f4xx.c

```diff
@@ -94,6 +94,10 @@ uint8_t timerClockDivisor(TIM_TypeDef *tim)
 
 uint32_t timerClock(TIM_TypeDef *tim)
 {
```

Comment:
Also - timerClockDivisor is affected too.. 
It would be best to use:
```
uint32_t timerClock(TIM_TypeDef *tim)
{
    return SystemCoreClock / timerClockDivisor;
}